help/en/Acknowledgements.shtml

```diff
@@ -104,6 +104,7 @@
         general has made more contributions than can be
         counted.</li>
 
+        <li>Daniel Bergqvist, who 
```

Comment:
Is this missing something?

Reply:
Thanks for catching that, forgot to save.